server/player/player.go

```diff
@@ -1728,7 +1733,7 @@ func (p *Player) checkBlockCollisions() {
 	w := p.World()
 
 	aabb := p.AABB().Translate(p.Position())
-	grown := aabb.Grow(0.25)
+	grown := aabb.GrowVec3(mgl64.Vec3{0, 0.25})
```

Comment:
Maybe this 0.25 needs to be changed. This is a rather big extension and I don't think we really need that. Maybe just 0.01 would even be enough.

Reply:
I'll try to see if that works too.